includes/modules/payment/paypalwpp.php

```diff
@@ -686,6 +686,9 @@ function install() {
 
     $db->Execute("insert into " . TABLE_CONFIGURATION . " (configuration_title, configuration_key, configuration_value, configuration_description, configuration_group_id, sort_order, date_added) values ('PayPal Merchant ID', 'MODULE_PAYMENT_PAYPALWPP_MERCHANTID', '', 'Enter your PayPal Merchant ID here. This is used for the more user-friendly In-Context checkout mode. You can obtain this value by going to your PayPal account, clicking on Profile and navigating to the My Business Info section; You will find your Merchant Account ID on that screen. A typical merchantID looks like FDEFDEFDEFDE11.', '6', '25', now())");
     $db->Execute("insert into " . TABLE_CONFIGURATION . " (configuration_title, configuration_key, configuration_value, configuration_description, configuration_group_id, sort_order, set_function, date_added) values ('Use InContext Checkout?', 'MODULE_PAYMENT_PAYPALWPP_CHECKOUTSTYLE', 'InContext', 'PayPal now offers a newer friendlier InContext (in-page) checkout mode (Requires that you enter your MerchantID in the Merchant ID Setting above). Or you can use the older checkout style which offers Pay Without Account by default but with a full-page-redirect.', '6', '25', 'zen_cfg_select_option(array(\'InContext\', \'Old\'), ', now())");
+    $db->Execute("insert into " . TABLE_CONFIGURATION . " (configuration_title, configuration_key, configuration_value, configuration_description, configuration_group_id, sort_order, set_function, date_added) values ('InContext Button Color?', 'MODULE_PAYMENT_PAYPALWPP_INCONTEXTCOLOR', 'gold', 'When PayPal\'s newer friendlier InContext (in-page) checkout mode is enabled PayPal offers different colors for the PayPal button. Select the color of the button to use.', '6', '25', 'zen_cfg_select_option(array(\'gold\', \'blue\', \'silver\'), ', now())");
+    $db->Execute("insert into " . TABLE_CONFIGURATION . " (configuration_title, configuration_key, configuration_value, configuration_description, configuration_group_id, sort_order, set_function, date_added) values ('InContext Button Size?', 'MODULE_PAYMENT_PAYPALWPP_INCONTEXTSIZE', 'small', 'When PayPal\'s newer friendlier InContext (in-page) checkout mode is enabled PayPal offers various PayPal button sizes. Select the size of the button to use.', '6', '25', 'zen_cfg_select_option(array(\'small\', \'tiny\', \'medium\'), ', now())"); 
+    $db->Execute("insert into " . TABLE_CONFIGURATION . " (configuration_title, configuration_key, configuration_value, configuration_description, configuration_group_id, sort_order, set_function, date_added) values ('InContext Button Shape?', 'MODULE_PAYMENT_PAYPALWPP_INCONTEXTSHAPE', 'small', 'When PayPal\'s newer friendlier InContext (in-page) checkout mode is enabled PayPal the PayPal button can take different shapes. Select the button shape to use.', '6', '25', 'zen_cfg_select_option(array(\'pill\', \'rect\'), ', now())"); 
```

Comment:
I "get" why you've done this. 
I'm curious: what's the likelihood of people needing to customize this? What's your gut say? 1 in 5?, 1 in 25? or 1 in 500?

(I'm mindful that we used to offer color customization options this way, but later moved them to the language file because nobody used them, and then later even PayPal deprecated and abandoned them completely.)